python/ray/train/_internal/stats.py

```diff
@@ -0,0 +1,117 @@
+import logging
+import os
+import threading
+from typing import Dict, Optional
+
+import ray
+from ray.data import Dataset
+from ray.train._internal.schema import TrainDatasetInfo, TrainRunInfo, TrainWorkerInfo
+from ray.train._internal.utils import check_for_failure
+from ray.train._internal.worker_group import WorkerGroup
+
+logger = logging.getLogger(__name__)
+
+
+@ray.remote(num_cpus=0)
+class TrainStatsActor:
```

Comment:
Reading over this, I realized there are no stats, and this points to a bunch of `Train...Info` objects. I wonder if there is a word that we can use instead of `Stats`/`Info` that captures/unifies these two.

Reply:
Yes, currently there's only static metadata stored in this actor, and in the future there will be stats.

What about `State`?